test/Prompt/DontWrap.C

```diff
@@ -172,7 +172,7 @@ const Test2::Nested::Value::type & Test2::Nested::reference() const {
 Test2::Nested Nest2(80);
 // CHECK: Nested::Nested(160)
 Nest2.simpleAdd(3)
-// CHECK: (Test2::Nested::Value::type) 83
+// CHECK: (int) 83
```

Comment:
I prefer `std::vector<int>::iterator` over the implementation detail.

But I prefer `int` over `std::vector<int>::value_type`.

So we cannot prefer one over the other. My suggestion: use the shortest name.